GUI/src/main/kotlin/models/AppState.kt

```diff
@@ -17,7 +17,10 @@ data class AppState(
     var video1Path: String = getPath("testVideo1.mkv"),
     var video2Path: String = getPath("testVideo2.mkv"),
     var outputPath: String = getPath("output.mkv"),
-    var sequenceObj: Array<AlignmentElement> = arrayOf<AlignmentElement>(),
+    var sequenceObj: Array<AlignmentElement> = arrayOf(),
+    var maskPath: String = getPath("mask.png"),
+    var gapOpenPenalty: Double = 0.3,
```

Comment:
These defaults are different from what was determined in https://github.com/amosproj/amos2023ws03-gui-frame-diff/pull/137.